develop/buildkite/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM temporalio/base-ci-builder:1.10.3 AS prebuild
+FROM temporalio/base-ci-builder:1.10.6 AS prebuild
+RUN apk add --update --no-cache sed
```

Comment:
can we put this in base-ci-builder?

Reply:
Can do. I'll update here once that's plumbed through

Reply:
https://github.com/temporalio/docker-builds/pull/167

Reply:
Merged and committed here